drmhwctwo.cpp

```diff
@@ -727,50 +746,82 @@ HWC2::Error DrmHwcTwo::HwcDisplay::SetVsyncEnabled(int32_t enabled) {
   return HWC2::Error::None;
 }
 
+//check whether the layer with the lease two zorder is the video layer
+bool DrmHwcTwo::HwcDisplay::WhetherVideoLayer(std::map< uint32_t,
```

Comment:
Is better to name as "IsXXXX" or "ContainXXX" for the method which return bool

Reply:
> 
> 
> Is better to name as "IsXXXX" or "ContainXXX" for the method which return bool

Get it.I'll change.